nexus/src/db/pool.rs

```diff
@@ -69,6 +70,12 @@ impl Pool {
     }
 }
 
+const DISALLOW_FULL_TABLE_SCAN_SQL: &str =
+    "set disallow_full_table_scans = on; set large_full_scan_rows = 0;";
+#[cfg(test)]
```

Comment:
```suggestion
/// SQL used to enable full table scans for the duration of the current transaction
///
/// We normally disallow table scans in effort to identify scalability issues during development.  But it's preferable for some ad hoc test-only queries to do table scans (rather than add indexes that are only used for the test suite).
///
/// This SQL appears to have no effect when used outside of a transaction.  That's intentional.  We do not want to use `SET` (rather than `SET LOCAL`) here because that would change the behavior for any code that happens to use the same pooled connection after this SQL gets run.
#[cfg(test)]
```

Reply:
Done in 653479b2d2b385a553a4326b6cede46dc8c094db